manifests/gcp_marketplace/guide.md

```diff
@@ -0,0 +1,86 @@
+# Deploy Kubeflow Pipelines from Google Cloud Marketplace
+
+You can go to [Google Cloud Marketplace](https://console.cloud.google.com/marketplace) to deploy Kubeflow Pipelines via UI.
+To get to Kubeflow Pipelines page in Marketplace, enter "Kubeflow Pipelines" in the search box in Marketplace landing page. 
+Alternatively you can access the [Marketplace page for Kubeflow Pipelines](https://console.cloud.google.com/marketplace/details/google-cloud-ai-platform/kubeflow-pipelines) directly.
+
+You can find Kubeflow Pipelines instances deployed via Marketplace in [Pipelines Console](http://console.cloud.google.com/ai-platform/pipelines).
+
+## Cluster
+
+You can choose an existing cluster for Kubeflow Pipelines deployment, or create a new cluster. New cluster created from Marketplace config page cannot be customized.
+If you want to customize the cluster, please go to [Google Kubernetes Engine](https://console.cloud.google.com/kubernetes/list) to create one.
+
+Please notice that you can only `install one Kubeflow Pipelines into one cluster`.
+
+## Namespace
+Please specify a [Kubenetes namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/).
+
+## App instance name
+It is possible to install multiple instances of Kubeflow Pipelines in a single cluster.
+This name will show up in [Pipelines Console](http://pantheon.corp.google.com/ai-platform/pipelines), so please use a user friendly name.
+
+## GCP Service Account credentials
+This credential is used to access GCS and Cloud SQL for managed storage option. 
+Also, if you pipeline need to access other GCP services (e.g. Dataflow), you will need to configure this credential.
+
+This deployment requires a service account to use for authentication when calling other GCP services. 
+Specify the base64-encoded credentials for the service account you want to use. 
+You can get these credentials by running the following command in a terminal window. 
+
+```
+$ gcloud iam service-accounts keys create application_default_credentials.json --iam-account [your-service-account] && cat application_default_credentials.json | base64
+```
+
+This command will create a new key under the service account. Please note that a single service account can only have 10 keys. Exising key also can be used.
```

Comment:
If you are going to say that an existing key can also be used, you should provide a parallel command showing how to do that.